includes/class-wc-dintero-hp-helper.php

```diff
@@ -101,6 +101,10 @@ public function convert_to_dintero_discounts($applied_coupons) {
 		return $discount_codes;
 	}
 
+	public function url_to_store_id($url) {
+		return str_replace( array( 'http://', 'https://'), array( '', '' ), $url);
+	}
```

Comment:
would it make sense to remove `www` after protocol and trailing `/` also?

Not sure if it is needed, but I would assume that a merchant could possibly register their site as something like `https://www.some-store.com/`.

Reply:
Agree about www. 

There is no trailing `/` in my tests, but I will make sure it's removed if it appears in other stores.